### PR TITLE
data: aksh-kr

### PR DESCRIPTION
**Pull request by aksh-kr**
adding new user aksh-kr

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3331"></a>

### CODE REVIEW

File: data/aksh-kr.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/akaskr",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https://resilientbloke.hashnode.dev",
+      "icon": "Fahashnode"
```
Comment (ChinmayMhatre) [inline data/aksh-kr.json:26]: This icon doesn't seem to be available in the Linkfree docs. I think it will be better is you use `SiHashnode`

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/aksh-kr.json (hunk 1 above)
Comment (aksh-kr) [reply data/aksh-kr.json]: Thank you Chinmay, 
I've changed the icon for hashnode and committed the changes.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by aksh-kr** [comment]
Thanks a lot, Pradumna!

**Pull request merged**